### PR TITLE
AT_01.01_42 | <Header> Head Icon is clickable and redirects to homepage

### PR DESCRIPTION
**Pull request by Olga975**
https://trello.com/c/tK3wexR2/1201-at010142-header-head-icon-is-clickable-and-redirects-to-homepage

### CODE REVIEW

File: cypress/e2e/headerHeadIcon.cy.js [hunk 1]
```diff
@@ -84,5 +84,11 @@ describe('Header Head Icon', () => {
           .should('have.prop', 'offsetLeft', 0)
     })
 
+    it('AT_01.01_42 | <Header> Head Icon is clickable and redirects to homepage', () => {
+        cy.get('[href="/asynchPeople/').click()
+        cy.get('#jenkins-head-icon').should('be.visible').click()
+        cy.get('.empty-state-block > h1').should('contain', 'Welcome to Jenkins')
```
Comment (DariaMas7) [inline cypress/e2e/headerHeadIcon.cy.js:90]: Welcome to Jenkins should be moved to fixtures

**Review by DariaMas7 — COMMENTED** [review]
I have one notice regarding the last line in your it() in case if my account already has a job created .empty-state-block > h1 locator and assert will not work
So it is better either to write in preconditions that there are no jobs created or change this line with assert

**Comment by Olga975** [comment]
> there are no jobs created

Thank you! Added this precondition to my trello TC card

File: cypress/e2e/headerHeadIcon.cy.js (hunk 1 above)
Comment (Olga975) [reply cypress/e2e/headerHeadIcon.cy.js]: > Welcome to Jenkins should be moved to fixtures

yes indeed! Also, added no jobs created precondition to my trello card. Thank you again!

**Review by SvetlanaMag — APPROVED** [review]

**Review by S-Lanchik — APPROVED** [review]

**Pull request merged**